src/components/accounts/TransactionBuilderProvider.tsx

```diff
@@ -1,27 +1,40 @@
-import React from 'react';
-
-import { TransactionBuilderContext } from "./TransactionBuilderContext";
+import { useState } from "react";
 
 import { Utxo } from '../../models/Utxo';
 
+import { TransactionBuilderContext } from "./TransactionBuilderContext";
+
 export const TransactionBuilderProvider = ({ children }) => {
   
-  const [inputs, setInputs] = React.useState<Map<string, Utxo>>(new Map<string, Utxo>());
+  // using object indirection so assignment creates new container object and forces re-render
+  const [inputsMap, setInputsMap] = useState<{ map: Map<string, Utxo> }>({ map: new Map<string, Utxo>() });
 
-  const getOutpoint = (utxo: Utxo) => `${utxo.txid}:${utxo.vout}`;
+  const getOutpoint = (utxo: Utxo) => `${utxo?.txid}:${utxo?.vout}`;
```

Comment:
we can add this function to the context values as well. Looks like we are using it in a lot of components and then replace the duplicate code in UtxoItem and AccountUtxoList component
I'm not sure Outpoint is a good choice for this variable, maybe getUtxoKey

Reply:
I've exposed and used the function through TransactionBuilderContext.

The spot where I'm rendering txid:vout in UtxoItem will be replaced with more UTXO details when I further work on the UTXO listing screen.

As to "Outpoint" name, I was just following what its called in bitcoin:
https://github.com/bitcoinbook/bitcoinbook/blob/develop/ch06_transactions.adoc#outpoint
https://developer.bitcoin.org/reference/transactions.html#outpoint-the-specific-part-of-a-specific-output